src/ptvsd/_vendored/pydevd/pydevd.py

```diff
@@ -216,7 +216,7 @@ def wait_pydb_threads_to_finish(self, timeout=0.5):
         while time.time() < started_at + timeout:
             if len(pydb_daemon_threads) == 1 and pydb_daemon_threads.get(self, None):
                 return
-            time.sleep(0.01)
+            time.sleep(1 / 30.)
```

Comment:
how about adding a comment on what the delay is change from 0.01 to ~0.03? it would be helpful for us to understand the rational behind this waiting time.

Reply:
That's really minor... my rationale is that 30 times/second would be ok and already 'real time' as far as the user is concerned -- if we don't have the finish condition we have to wait for other threads so, trying 100 times/second seemed a bit too much for me (I didn't add a comment because that seems very minor for me -- trying 100 times/second would not be terribly bad either.